apps/src/dance/DanceMetricsReporter.ts

```diff
@@ -0,0 +1,5 @@
+import {LabMetricsReporter} from '../lab2/Lab2MetricsReporter';
+
+const DanceMetricsReporter = new LabMetricsReporter({appName: 'Dance'});
+
+export default DanceMetricsReporter;
```

Comment:
This is a little confusing to me...since we're importing a `LabMetricsReporter` _class_ to construct a `DanceMetricsReporter` _object_.

Could this be renamed something like `LabMetrcisReporterConstructor` to better differentiate?

Reply:
Hmm, gotcha. I personally prefer to keep the import name the same as the class...would doing something like
```
const danceReporterInstance = new LabMetricsReporter({appName: 'Dance'});

export default danceReporterInstance as DanceMetricsReporter;
```
be clearer?

Reply:
Discussed offline, will change the export to lowercase (`danceMetricsReporter`) and keep the class name

Reply:
Thanks! Just curious - when you create a singleton of `LabMetricsReporter` in the `Lab2Registry` will you use lower or uppercase?

Reply:
Uppercase for the import and lowercase for the field name. Maybe calling it a singleton is a bit misleading - it would just be a field on the `Lab2Registry` class, but because `Lab2Registry` is a singleton, its metrics reporter field can also be treated like a singleton. Basically the same as project manager: https://github.com/code-dot-org/code-dot-org/blob/11012fd30f01eae878ae3e57f8007fa7f34e6627/apps/src/lab2/Lab2Registry.ts#L35-L37